### DISCUSSION TITLE
How to disable/hide external wallet options? - @web3auth/web3auth

### DISCUSSION THREAD

**Discussion by qqqllppp** [Web3Auth/Web3Auth#543] [Help]
How to disable certain external wallet options? For example, I want to disable/hide `WalletConnect` from my users. I am using `@web3auth/web3auth`.

![Screenshot from 2022-09-05 10-47-05](https://user-images.githubusercontent.com/96459097/188351131-672d0d4d-71b5-4d31-8886-05480687da84.png)

**Comment by himanshuchawla009** [comment]
Here is the code snippet to hide wallets from modal.
```
await web3auth.initModal({
  modalConfig: {
    [WALLET_ADAPTERS.WALLET_CONNECT_V1]: {
      showOnModal: false
    },
  },
});

```

Refer to this section for more info: https://web3auth.io/docs/sdk/web/web3auth/initialize#arguments-1
